dora/core/server/worker/src/main/java/alluxio/worker/dora/DoraOpenFileHandleContainer.java

```diff
@@ -40,14 +40,19 @@ public class DoraOpenFileHandleContainer extends Thread {
   public void run() {
     while (!mStop) {
       try {
-        sleep(Constants.MINUTE_MS);
+        sleep(Constants.MINUTE_MS * 10);
+        if (mStop) {
+          LOG.debug("Thread exited");
+          return;
+        }
+        LOG.debug("Checking all handles ...");
         // Iterate the mOpenFileHandles if some handles are stale (not active for a long time).
         for (Map.Entry<String, OpenFileHandle> entry : mOpenFileHandles.entrySet()) {
           String key = entry.getKey();
           OpenFileHandle handle = entry.getValue();
-          if (System.currentTimeMillis() - handle.getLastAccessTimeMs() >= Constants.HOUR) {
-            mOpenFileHandles.remove(key);
-            handle.close();
+          LOG.debug("Checking handle for {}", handle.getPath());
+          if (System.currentTimeMillis() - handle.getLastAccessTimeMs() >= Constants.HOUR * 24) {
```

Comment:
i'm thinking if we should make this configurable

Reply:
With this patch, we are not going to close the handle. It only reports warning message.
I don't want to add configuration keys at this moment. We have had too many keys.

When we really to cleanup the stale open handles, we may add configurable keys. That is a future task.
To enable clean up stable open handles, we need connection/session support.
When a session is ended/terminated, all the open handles in this session should be cleaned up.